Library/Homebrew/download_strategy.rb

```diff
@@ -665,6 +665,11 @@ def initialize(name, resource)
     @url = @url.sub("svn+http://", "")
   end
 
+  def cache_filename
+    # "@" is a special character for Subversion. Keep it out of the cache dir name.
+    "#{name.gsub("@", "_AT_")}--#{cache_tag}"
```

Comment:
Do you think we need the `_`? I also wonder about using `-` instead.

Reply:
Switched to `-`. I like having some separator there for readability.